api/pb/android/threads/src/main/java/io/textile/threads/Client.java

```diff
@@ -83,13 +102,29 @@ public void StartFromAddressSync (String storeID, String address, ByteString fol
         return;
     }
 
+    public void StartFromAddress (String storeID, String address, ByteString followKey, ByteString readKey, StreamObserver<StartFromAddressReply> responseObserver) {
+        StartFromAddressRequest.Builder request = StartFromAddressRequest.newBuilder();
+        request.setStoreID(storeID);
+        request.setAddress(address);
+        request.setFollowKey(followKey);
+        request.setReadKey(readKey);
+        asyncStub.startFromAddress(request.build(), responseObserver);
+    }
+
 
     public GetStoreLinkReply GetStoreLinkSync (String storeID) {
         GetStoreLinkRequest.Builder request = GetStoreLinkRequest.newBuilder();
         request.setStoreID(storeID);
         return blockingStub.getStoreLink(request.build());
     }
 
+
+    public void GetStoreLink (String storeID, StreamObserver<GetStoreLinkReply> responseObserver) {
```

Comment:
maybe add a comment that this method won't work until you do a `client.start()`.

Reply:
holding off until fixed in threads to not need it (coming up)